src/data/achievements.js

```diff
@@ -241,6 +241,33 @@ const achievements = [
       inventoryLimit: state.inventoryLimit + reward,
     }),
   }))(),
+
+  ((goal = 500000) => ({
+    id: 'i-am-rich-1',
+    name: 'I am Rich!',
+    description: `Earn ${dollarString(goal)}`,
+    rewardDescription: 'All sales get a 5% bonus',
```

Comment:
makes sense! i made a hash of `I_AM_RICH_BONUSES`, the keys line up to the # used in the 'id', which I thought was simpler to understand than just an array, but am open to whatever you prefer here.

Reply:
I noticed this reply late, but I addressed it in https://github.com/jeremyckahn/farmhand/pull/77/files#r641982208, but yes I (personally) think it would a little more intuitive for this to be an `Array`. 🙂

Reply:
sounds good! going to resolve this conversation since that one exists now